styleguide/_themes/derek/scss/components/mainnav.scss

```diff
@@ -0,0 +1,354 @@
+//main nav
+.main-nav {
+  background: $dark-gray;
+  border: 0;
+  border-radius: 0;
+  margin: 0;
+
+  .navbar-menuContainer {
+    border: 0;
+    max-height: none;
+    padding: 0;
+  }
+}
+
+.navbar-nav {
```

Comment:
In this case I suppose it could be worse.

With the sheer amount of styling and overrides happening in this PR I question the need for using bootstrap's `.navbar` classes altogether.